Project.toml

```diff
@@ -4,6 +4,7 @@ authors = ["Arpit Bhatia <arpit16229@iiitd.ac.in>"]
 version = "0.1.0"
 
 [deps]
+CSDP = "0a46da34-8e4b-519e-b418-48813639ff34"
```

Comment:
Compat bounds for CSDP?

Reply:
CSDP is a bad dependency to have because it does not build on windows.
Can you use SCS?
Also this means we should be testing tutorial in all OSs.